### ISSUE TITLE
gpu usage during inference

### ISSUE THREAD

**Issue by kavya2425** [Open-Speech-EkStep/vakyansh-tts#7]
Hi 

is it compulsory to use GPU during inference while using your models.

**Comment by ananthu-sk** [comment]
No, [see this](https://github.com/Open-Speech-EkStep/vakyansh-tts/blob/4612b386e0e604fe892ec03e8cb1227eabc6080c/tts_infer/example_inference.py#L24).
